docs/store/starling.md

```diff
@@ -12,61 +12,92 @@ breadcrumb: 'Starling'
 <a href="https://starlingstorage.io" target="_blank"><img src="./images/starling.gif" alt="starling" /></a>
 </center>
 
-## Get started with Starling
+## Getting started
 
-1. Starling uses [Lotus](../get-started/lotus/README.md) to interact with the Filecoin network. [download, install and run a Lotus daemon](../get-started/lotus/installation.md):
+This guide will quickly set up Starling on your computer. For more information on Starling and how to use it, check out the [official Starling documentation](https://starlingstorage.io/).
 
-   Make sure that Lotus is running and the find out about your Lotus API token and endpoint with:
+### Prerequisites
 
-   ```sh
-   lotus auth api-info --perm admin
-   ```
+You need to have a couple of things installed before you can interact with Starling:
 
-   This will print a `FULLNODE_API_INFO=<token>:/ip4/127.0.0.1/tcp/<port>/http` string that we will use later.
+1. [Lotus](../get-started/lotus/installation.md).
+1. [NodeJS](https://nodejs.org/en/download/)
 
-2. [Install NodeJS](https://nodejs.org/en/download/):
+### Steps
 
-   You will need a recent version of nodejs and npm (>=2.19).
```

Comment:
This information got lost, and it is important, given some distros come by default with older versions of nodejs/npm.

Reply:
Should this be NodeJS 12.19? I can't find a `2.19` version of either [NodeJS](https://nodejs.org/en/download/releases/) or NPM(https://www.npmjs.com/package/npm?activeTab=versions).

Reply:
I think yes, I made a typo :(